backend/repositories/match-repository.js

```diff
@@ -22,8 +22,32 @@ const storeMatch = async (match) => {
     try {
         result = await storageContext.insertMatch(match)
 
-        await updateRatingForTeam(winningTeam, match.ratingChange, storageContext)
-        await updateRatingForTeam(losingTeam, -match.ratingChange, storageContext)
+        // FIXME: foosbot's report of rating change is incorrect if the number of players on each side is not the same
+        // one player beat two players
+        if (winningTeam.length < losingTeam.length) {
+            // he gains rating from both opponents
```

Comment:
Please refrain from using gender specific pronouns when referring to users/players. Either use `a player gains` or https://en.wikipedia.org/wiki/Singular_they

Reply:
good catch!

Reply:
Great catch indeed, I can't imagine what could have happened if something this serious slipped into our production code 😱.

Reply:
Yeah, I purposefuly decided to go with a commonly used error in order not to use "they" for both the two players and the single one. Thought it would be more clear. And once again, I didn't think you'd take it this seriously. But sure, I'll change it. One day...